src/sentry/integrations/message_builder.py

```diff
@@ -51,8 +51,6 @@ def format_actor_option(actor: Team | RpcUser, use_block_kit: bool = False) -> M
             }
         return {"text": f"#{actor.slug}", "value": f"team:{actor.id}"}
 
-    raise NotImplementedError
```

Comment:
I think I would probably change the `if`s to `elifs` and `else: raise AssertionError('unreachable')`